[Docs] Clarify type of Main Node in tutorial. (#4072)
godotengine/godot-docs: getting_started/step_by_step/your_first_game.rst

----------

Now it's time to bring it all together. Create a new scene and add a

> :ref:`Node <class_Node>` named ``Main``. This should be a basic Node, NOT a Node2D. Click the "Instance" button and select your

Reviewer: ```suggestion
:ref:`Node <class_Node>` named ``Main``. Ensure you create a Node, **not** a
Node2D. Click the "Instance" button and select your
```
